app/renderer/components/preferences/helpfulHints.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const ImmutableComponent = require('../../../../js/components/immutableComponent')
+const {StyleSheet, css} = require('aphrodite')
+
+class HelpfulHints extends ImmutableComponent {
+  render () {
+    return <div className={css(styles.helpfulHints)}>
+      <span className={css(styles.hintsTitleContainer)}>
+        <span className={css(styles.hintsTitle)} data-l10n-id='hintsTitle' />
+        <span className='fa fa-refresh' onClick={this.props.refreshHint}
+          style={{
+            color: 'white',
+            marginLeft: '15px',
+            fontSize: 'smaller'
+          }}
+        />
+      </span>
+      <div className={css(styles.hintsNumber)} data-l10n-id={`hint${this.props.hintNumber}`} />
+      <div className={css(styles.helpfulHintsBottom)}>
+        <a className={css(styles.helpfulHintsBottomLink)}
+          target='_blank' href='https://community.brave.com/'
+          data-l10n-id='submitFeedback' />
+      </div>
+    </div>
+  }
+}
+
+const styles = StyleSheet.create({
+  helpfulHints: {
+    display: 'block',
+    width: '100%',
+    color: '#eee',
```

Comment:
I know this comes from the original code but I thought making this `white` and setting `#eee` to elements which requires it should be better.

But `#eee` is not used anywhere so this can be just replaced with `white`.